action.yml

```diff
@@ -9,7 +9,7 @@ inputs:
   python-version:
     description: "Python version that will be deployed"
     required: false
-    default: "3.10"
+    default: "3.11"
```

Comment:
99% of the projects this is used for are Nautobot related... Nautobot does not currently support 3.11.  I'm opposed to changing the default until that time